src/kirby/components/list/list.component.html

```diff
@@ -17,6 +17,7 @@
         <ion-item *ngFor="let item of section.items; let idx = index;trackBy: trackByFn"
                   [class.has-divider]="showDivider"
                   [class.selectable]="isSelectable"
+                  [class.selected]="markSelectedRow && selectedItem && item.id === selectedItem.id"
```

Comment:
You can not take it for granted that the items handed to the kirby-list has an `id` property. So you shouldn't make use of any properties on the items.

You will have to find an other way to identify the selected item.

Reply:
Fixed this by adding dynamic _id guid